doc/ci_admin_guide/Reconnecting_Host.md

```diff
@@ -0,0 +1,84 @@
+# Reconnecting Host to Contrail Windows CI cluster
+
+This document describes a procedure required to reconnected a VMware ESXi host to the Contrail Windows CI cluster.
+Reconnecting a host to CI cluster is understood as a set to steps leading to usage of this host in Contrail Windows CI.
+
+Described procedure must be performed only when:
+
+- a host with reinstalled ESXi must be reconnected to the cluster,
+- a host with replaced hard drive must be reconnected to the cluster.
+
+## Prerequisites
+
+Requirements for CI admin:
+
+- Access to Contrail Windows CI infrastructure.
+- Credentials to Contrail Windows CI VMware cluster.
+- Browser with Flash support, preferably Chrome.
+
+Requirements for a reconnected host:
+
+- Host must be connected to vCenter.
+
+## Steps
+
+1. Open the vSphere Web Client and login using credentials provided by Contrail Windows CI team.
+1. In `Navigator` pane select `Hosts and clusters` tab.
+1. Expand `CI-DC` datacenter entry.
+1. Expand `WinCI` cluster entry.
+1. Find a host entry representing a being reconnected.
+    - If a host cannot be found, it is not connected to vCenter. In that case, please contact infrastructure team.
+1. If a host is marked as being in `maintenance mode`:
+    - Right click on a host entry and select `Maintenance mode > Exit maintenance mode`.
+1. In `Navigator` pane select `Storage` tab.
+    - Right click on `NFS-Datastore` and select `Mount Datastore to Additional Hosts`.
+    - Mark a checkbox next to a host entry representing a being reconnected.
+    - Click `OK`.
+    - Right click on `winci_nfsbackup` and select `Mount Datastore to Additional Hosts`.
+    - Mark a checkbox next to a host entry representing a being reconnected.
+    - Click `OK`.
+1. In `Navigator` pane select `Hosts and clusters` tab.
+1. Find a host entry representing a being reconnected and click on it.
+1. Cleanup of orphaned VMs must be performed.
+    - In the middle pane select `VMs` tab.
+    - In the VM table, click `Name` header to sort VMs by name in ascending order.
+    - For each VM perform a following process:
+        - If a VM is marked as `Orphaned` and it is not critical to CI functioning:
```

Comment:
We should link to list of VMs in the infra and how to determine if it's critical or not